onnxmltools/convert/main.py

```diff
@@ -21,12 +21,11 @@ def convert_keras(model, name=None, initial_types=None, doc_string='',
                   target_opset=None, targeted_onnx=onnx.__version__,
                   channel_first_inputs=None, custom_conversion_functions=None, custom_shape_calculators=None,
```

Comment:
Add a warning here to say the custom_conversion_XXX is not supported any more if this argument is not empty.